### PR TITLE
COMMERCE-12574

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://liferay.atlassian.net/browse/COMMERCE-12574

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:functional-commerce

**Comment by Brunno0liveira** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase [hunk 1]
```diff
@@ -284,6 +284,158 @@ definition {
...
+				key_value = "Order Split by Catalog Action",
+				locator1 = "CommerceEntry#HEALTH_CHECK_ICON_CHECK_NAME");
+
+			CommerceJSONOrdersAPI._patchCommerceOrderObjectAction(
+				active = "true",
+				objectActionName = "SplitOrderByCatalog",
+				objectDefinitionName = "CommerceOrder");
+		}
+
+		task ("And the buyer checkout an order") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase:376]: And the buyer creates an order with a single product

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase [hunk 2]
```diff
@@ -284,6 +284,158 @@ definition {
...
+			Click(
+				key_text = "Accept Order",
+				locator1 = "Link#ANY");
+
+			Alert.viewSuccessMessage();
+
+			CommerceOrders.openOrdersAdmin();
+		}
+
+		task ("Then the Order is splitted in two and the Admin can view both orders") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderSplitting.testcase:427]: Then admin can view both orders

**Review by baharturk — CHANGES_REQUESTED** [review]
Please take a look at my comments. Thanks!

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
ci:test:stable

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:reevaluate:1397401_8031

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:reevaluate:1257401_3963

**Comment by baharturk** [comment]
ci:test:stable

**Comment by Brunno0liveira** [comment]
ci:test:stable

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
The issues are not related to the changes, PR can be forwarded.

@victoroscuro

**Comment by victoroscuro** [comment]
ci:forward:force

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**